pgmpy/models/FactorGraph.py

```diff
@@ -5,6 +5,7 @@
 
 import numpy as np
 from networkx.algorithms import bipartite
+import six
```

Comment:
@kris-singh Please do not import six like this. Import it from `pgmpy.extern`. And I think I have mentioned this in your previous PR also. Please be more careful about these things.